src/FormField/RegisteredMFAMethodListField.php

```diff
@@ -2,17 +2,43 @@
 
 namespace SilverStripe\MFA\FormField;
 
+use InvalidArgumentException;
 use SilverStripe\Admin\SecurityAdmin;
 use SilverStripe\Forms\FormField;
 use SilverStripe\MFA\Controller\AdminRegistrationController;
 use SilverStripe\MFA\Model\RegisteredMethod;
 use SilverStripe\MFA\Service\MethodRegistry;
 use SilverStripe\MFA\Service\RegisteredMethodManager;
 use SilverStripe\MFA\Service\SchemaGenerator;
+use SilverStripe\Security\Member;
 use SilverStripe\Security\Security;
 
 class RegisteredMFAMethodListField extends FormField
 {
+    /**
+     * Because {@see setValue()} can do nothing on e.g. form submission, it is of critical importance that this
+     * object is never in existence without a valid Member set as the value - we have set the `$value` parameter as
+     * required, however we still need to ensure that it is a valid value. We do this via type hints, and removing the
+     * optionality of the parameters for this constructor - excepting that one can still pass `null` for the $title.
```

Comment:
I don't think the last sentence needs to be there to be honest, the signature essentially says this already